soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -308,4 +326,31 @@ struct gpio_reg {
 	volatile uint8_t PLOCK_CTL;
 };
 
+/*
+ * Pulse Width Modulator (PWM) device registers
+ */
+struct pwm_reg {
+	/* 0x000: Clock Prescaler */
+	volatile uint16_t PRSC;
```

Comment:
My opinion of the upper-case register names has not improved

Reply:
Here is our discussion for this in the other PR. https://github.com/zephyrproject-rtos/zephyr/pull/27578#discussion_r474971026. Currently, it doesn't violate the code style of Zephyr. I still strongly suggest keeping them the same as our datasheet possibly. The other important reason is I want to keep the same style as our HAL did. Or it makes me  hard to maintain updates between Zephry and NPCX HAL drivers

Reply:
I think we should keep is consistent with other Nuvoton code for now. If we want to change it, we should change it all of the register definitions for Nuvoton.